frontend/integration-tests/views/overview.view.ts

```diff
@@ -1,15 +1,15 @@
 import { $, $$, browser, by, element, ExpectedConditions as until } from 'protractor';
 
 export const projectOverview = $('.project-overview');
-const projectOverviewItemSelector = '.project-overview__item';
-export const projectOverviewListItems = $$(projectOverviewItemSelector);
 export const detailsSidebar = $('.overview__sidebar');
 export const detailsSidebarOverviewTab = detailsSidebar.element(by.buttonText('Overview'));
 export const detailsSidebarResourcesTab = detailsSidebar.element(by.buttonText('Resources'));
 export const detailsSidebarTitle = $('.resource-overview__heading .co-m-pane__name');
 
-export const itemsAreVisible = () => {
-  return browser.wait(until.presenceOf($(projectOverviewItemSelector)));
+export const testObjectVisible = (name) => {
+  return browser.wait(
+    until.presenceOf(element(by.cssContainingText('.project-overview__item-heading--name', name))),
```

Comment:
Don't use `cssContainerText`. Create a `data-test-id` if necessary. But I'm not sure we want this change to be honest (see my other comment in the scenario).

Reply:
Dropped, as FF worked fine with the original code (with `bridge` running on host).